packages/amplify-graphql-api-construct/src/types.ts

```diff
@@ -762,6 +763,19 @@ export interface AmplifyGraphqlApiProps {
    * For more information, refer to https://docs.amplify.aws/lib/datastore/getting-started/q/platform/js/
    */
   readonly dataStoreConfiguration?: DataStoreConfiguration;
+
+  /**
+   * Specifies the logging configuration when writing GraphQL operations and tracing to Amazon CloudWatch for an AWS AppSync
+   * GraphQL API.
+   *
+   * `logging: true` or `logging: {}` enables logging with default settings.
+   *
+   * Default values are:
+   * - excludeVerboseContent: true
+   * - fieldLogLevel: NONE
+   * - retention: ONE_WEEK
+   */
+  readonly logging?: true | LogConfig;
```

Comment:
As discussed, let's add more description about each of these options, not just the default values. In particular, we want to note that `excludeVerboseContent` will log the full incoming query including user parameters. We also want to advise customers that they should review their IAM policies to ensure that only authorized users can review their CloudWatch logs.

We should ALSO link to the [published docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appsync-graphqlapi-logconfig.html), but we want to ensure that customers see the advisory messages at point of use, not require them to navigate away.